src/sentences/gsa.rs

```diff
@@ -27,17 +32,49 @@ pub enum GsaMode2 {
 pub struct GsaData {
     pub mode1: GsaMode1,
     pub mode2: GsaMode2,
-    pub fix_sats_prn: Vec<u32>,
+    pub fix_sats_prn: Vec<u32, 12>,
     pub pdop: Option<f32>,
     pub hdop: Option<f32>,
     pub vdop: Option<f32>,
 }
 
-fn gsa_prn_fields_parse(i: &[u8]) -> IResult<&[u8], Vec<Option<u32>>> {
+/// This function is take from `nom`, see `nom::multi::many0`
+/// with one difference - we use a [`heapless::Vec`]
+/// because we want `no_std` & no `alloc`
+fn many0<I, O, E, F>(mut f: F) -> impl FnMut(I) -> IResult<I, Vec<O, 12>, E>
+where
+    I: Clone + InputLength,
+    F: Parser<I, O, E>,
+    E: ParseError<I>,
+    O: core::fmt::Debug,
+{
+    move |mut i: I| {
+        // let mut acc = crate::lib::std::vec::Vec::with_capacity(4);
```

Comment:
Dead code?

Reply:
Dead but this is the original nom code. I think it would be good to leave it as it is.